src/app/modules/myinfo/myinfo.controller.ts

```diff
@@ -202,6 +204,7 @@ export const loginToMyInfo: ControllerHandler<
  * MyInfo data.
  */
 export const handleMyInfoLogin = [
+  authCallbackForwardingMiddleware,
```

Comment:
Can this be on `src/app/modules/myinfo/myinfo.routes.ts` so that the auth middleware is consistently applied on all routes?

Reply:
yup, will apply the middleware consistently on `*.routes.ts`
```suggestion
```